SU-HSDO/suhumsci
# Paragraph Previews

# READY FOR REVIEW

## Summary
- Enable paragraph previews in all paragraph fields
- Create custom CSS for previews on `humsci_basic` subthemes
- Fixes in `humsci_basic` to avoid using hardcoded `rem` dimensions
- Add libraries in `su_humsci_gin_admin` theme, to correctly style previews
- Add templates for displaying paragraph previews in edit forms
- Move preprocess code from `humsci_basic` theme to modules (`hs_layouts` and `hs_field_helpers`) to make available to all themes

## Need Review By (Date)
09/11

## Urgency
medium

## Steps to Test
In both Colorful and Traditional:
1. Edit existing flexible and private pages, with different kinds of components. Confirm that you can see the preview for all existing kinds of components (except Developer's Catalog, where a placeholder is shown)
2. For components with nested paragraphs (like collections or slideshows), confirm that you can see a preview of the nested paragraphs
3. Confirm that there are no visual changes in the user facing pages

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
